pkg/sdn/plugin/node.go

```diff
@@ -270,24 +278,38 @@ func (node *OsdnNode) Start() error {
 		return err
 	}
 
+	var podsToKill []kapi.Pod
 	if networkChanged {
```

Comment:
we can only have podsToKill if the network changed so maybe all the killing code should be inside the if body?